scripts/skills/perks/perk_rf_iron_sights.nut

```diff
@@ -15,16 +15,30 @@ this.perk_rf_iron_sights <- ::inherit("scripts/skills/skill", {
 		this.m.IsHidden = false;
 	}
 
+	function isEnabled()
```

Comment:
Depending on the outcome of the tech discussion:
- Since this is a newly introduced function it should at least be sorted to the bottom, ideally with a comment introducing that section